habit/src/index.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
```

Comment:
Should we have done this outside src?
Cannot think of a reason to do it inside src

Reply:
As in, the src/index.js

Reply:
why not inside? src should have the main index file.